hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/ReadableAccountStore.java

```diff
@@ -38,6 +38,16 @@ public interface ReadableAccountStore {
     @Nullable
     Account getAccountById(@NonNull final AccountID accountID);
 
+    /**
+     * Fetches an {@link Account} object from state with the given alias. If the account could not be
+     * fetched because the given account doesn't exist, returns {@code null}.
+     *
+     * @param alias alias
+     * @return AccountID object if successfully fetched or {@code null} if the account doesn't exist
+     */
+    @Nullable
+    AccountID getAccountIDByAlias(@NonNull final String alias);
```

Comment:
@Neeharika-Sompalli or @tinker-michaelj is this signature correct? I thought an alias was a sequence of bytes, not a `String`

Reply:
yeah it is Bytes, but we stored it as String in states. Similar to `ByteString` in mono-service